Misc/NEWS.d/next/Build/2022-10-28-22-24-26.gh-issue-98831.IXRCRX.rst

```diff
@@ -1 +1,10 @@
-We have new tooling, in ``Tools/cases_generator``, to generate the interpreter switch from a list of opcode definitions.
+We have new tooling, in ``Tools/cases_generator``,
+to generate the interpreter switch from a list of opcode definitions.
+This affects only people who are adding or modifying instruction definitions.
```

Comment:
```suggestion
This only affects adding, modifying or removing instruction definitions.
```

* It sounds a little strange to frame it in terms of "people" in a changelog, as opposed to actions
* Doesn't it affect removing opcodes too?